crates/jstzd/tests/utils.rs

```diff
@@ -13,3 +13,13 @@ where
     }
     false
 }
+
+#[allow(dead_code)]
+pub async fn get_request(endpoint: &str) -> String {
+    reqwest::get(endpoint)
+        .await
+        .expect("Failed to get block head")
```

Comment:
Maybe change this error message here since this sends a general get request and is in utils now?